src/config/vscode_config.json

```diff
@@ -1,11 +1,12 @@
 {
-  "ide_path": "",
+  "ide_path": "/usr/bin/code",
   "plugin_cache_path": "",
   "project_paths" :
```

Comment:
We can create a separate json called analysis_data.json which will be used by all IDEs  and use it through a fixture 


Sample analysis_data.json


    {
        "thorntail_to_eapxp": {
        "targets": [ "eapxp" ],
        "story_points": "abcd.png"
      },
        "weblogic_to_eap7": {
        "targets": ["eap7" ],
        "story_points": "xyz.png"
         }
      }


Fixture : 

     `
    @pytest.fixture(scope="session")
    def analyses_data():
         with open("analysis_data.json", "r") as file:
              return json.load(file)`


And then you call this fixture in your test.

target = analysis_data["thorntail_to_eapxp][target]
story_points= analysis_data["thorntail_to_eapxp"][story_points]